src/version.js

```diff
@@ -8,7 +8,13 @@ function trackNewVersion({
   newRange,
   newVersion,
 }) {
+  // wildcards don't need to be changed
+  if (oldRange === '*') {
```

Comment:
Can you figure out a way to go through `semver.Range`? This doesn't support strings like " * " or "" (which are also valid).

Reply:
oh! I didn't know that'd be valid! silly :)

Reply:
I don't think those are valid actually

Reply:
They look fine to me.

Reply:
semver.valid is for versions, not ranges.